core/src/main/resources/hudson/model/View/newJob.jelly

```diff
@@ -22,45 +22,52 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 -->
 
-<!--
-  "New Project" page.
--->
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:s="/lib/form">
   <j:getStatic var="permission" className="hudson.model.Item" field="CREATE"/>
 
-  <l:layout norefresh="true" cssclass="add-item main-panel-only" permission="${permission}" title="${%NewJob(it.newPronoun)}">
+  <l:layout norefresh="true" cssclass="main-panel-only" permission="${permission}" title="${%NewJob(it.newPronoun)}">
 
     <l:js src="jsbundles/add-item.js" />
     <l:css src="jsbundles/add-item.css" />
 
     <l:main-panel>
-      <div id="create-item-panel" style="display: none;">
-      <s:form method="post" action="createItem" name="createItem">
+      <div id="add-item-panel" style="display: none;">
+        <form method="post" action="createItem" name="createItem" id="createItem">
+          <div class="header">
+            <div class="add-item-name">
+              <label for="name">${%ItemName.label}</label>
+              <input name="name" id="name" type="text" />
+              <div class="input-help">&#187; ${%ItemName.help}</div>
```

Comment:
BTW these sources are guaranteed to be UTF-8, so you can just use `»` directly—easier to read.